index.js

```diff
@@ -235,33 +234,35 @@ function schemaMiddleware (schema) {
 }
 
 function cors (opts) {
-  var _cors = corsify(opts)
+  opts = opts || {}
+  var headers = opts.headers || ['Content-Type', 'Accept', 'X-Requested-With']
+  var methods = opts.methods || ['PUT', 'POST', 'DELETE', 'GET', 'OPTIONS']
+  var credentials = opts.credentials || true
+  var origin = opts.origin || '*'
+
   return function (handler) {
-    var obj = {}
+    var _handler = null
     if (typeof handler === 'object') {
       var keys = Object.keys(handler)
 
       assert.equal(keys.length, 1, 'merry.cors: we can only corsify a single method per endpoint')
```

Comment:
Also I'm not sure how much sense it makes to allow multiple methods, but then only handle a single type of handler. I feel something's not right here. How do we want to approach this?